csharp_dotnetcore/60.ConversationHistory/appsettings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "ConnectionString":  "The connection string to the BLOB store"
+}
```

Comment:
Consider renaming the variable to something better, just as "AzureBlobConnectionString" or similar.